api-specs/openrpc-user-api.json

```diff
@@ -553,6 +553,28 @@
                 "type": "string",
                 "description": "The unique identifier of the command associated with the transaction."
             },
+            "VerifiedNetwork": {
+                "title": "VerifiedNetwork",
+                "description": "Structure representing the Networks with connectivity verification",
+                "allOf": [
+                    {
+                        "$ref": "#/components/schemas/Network"
+                    },
+                    {
+                        "title": "VerifiedNetworkProperties",
+                        "description": "holder for boolean verified property",
+                        "type": "object",
+                        "properties": {
+                            "verified": {
```

Comment:
very nit-picky, but I'm not sure how I feel about the name `verified` for this property. I think `active`, `live`, or `reachable` maybe gives the idea better?